x-pack/scripts/functional_tests.js

```diff
@@ -6,16 +6,16 @@
 
 require('@kbn/plugin-helpers').babelRegister();
 require('@kbn/test').runTestsCli([
-  require.resolve('../test/reporting/configs/chromium_api.js'),
-  require.resolve('../test/reporting/configs/chromium_functional.js'),
-  require.resolve('../test/reporting/configs/phantom_api.js'),
-  require.resolve('../test/reporting/configs/phantom_functional.js'),
-  require.resolve('../test/functional/config.js'),
+//  require.resolve('../test/reporting/configs/chromium_api.js'),
```

Comment:
consider adding a `This should not be merged into master...` comment (like the others) at the start of this section